packages/@adobe/spectrum-css-temp/components/button/index.css

```diff
@@ -115,12 +115,10 @@ governing permissions and limitations under the License.
                 fill var(--spectrum-global-animation-duration-100) ease-out;
     box-sizing: initial;
   }
-
 }
 
 .spectrum-Button {
-  @inherit: %spectrum-BaseButton;
-  @inherit: %spectrum-FocusRing;
+  composes: spectrum-BaseButton spectrum-FocusRing;
```

Comment:
At the moment it will be duplicated due to postcss-import. That runs before CSS modules get processed, so it'll look like a single file and thus get the same hash.